cmd/kubelet/app/server.go

```diff
@@ -344,6 +346,9 @@ func (s *KubeletServer) KubeletConfig() (*KubeletConfig, error) {
 	return &KubeletConfig{
 		Address:                   s.Address,
 		AllowPrivileged:           s.AllowPrivileged,
+		Authenticator:             nil, // default does not enforce auth[nz]
```

Comment:
I'm uncomfortable with silent defaulting to unanthenticated access.   Should we perhaps rename this function to "UnanthenticatedKubeletConfig()" or similar to make it clear?
I realize that silent defaulting to unanthenticated access does not represent a regression today, but I worry about future callers who might not assume/realize that they're getting a completely unauthenticated config by default.